### DISCUSSION TITLE
How to migrate XML parser options from v3 to v4 ?

### DISCUSSION THREAD

**Discussion by ningji** [NaturalIntelligence/fast-xml-parser#488] [Q&A]
For example i believe option ignoreNameSpace in v3 is replace with removeNSPrefix.
Is there a porting guide ?
Thanks !

**Comment by ningji** [comment]
Never mind, i found in the change log. But if there's a better place, pls let me know, thx !

**Comment by amitguptagwl** [comment]
Yes. Please refer changelogs
https://github.com/NaturalIntelligence/fast-xml-parser/blob/master/CHANGELOG.md
